### DISCUSSION TITLE
FluentUI react control for displaying bookmarks

### DISCUSSION THREAD

**Discussion by manjunathreddy1** [microsoft/fluentui#28678] [General] [answered]
Hi All,

Is there any FluentUI react control to display bookmarks as shown in the attached image. I didn't find it in react controls list. 

![image](https://github.com/microsoft/fluentui/assets/265137/03fe8dcb-dc80-45dc-b38f-11414b1afc02)

**Comment by ValentinaKozlova** [comment, marked as answer]
Yes, within the Fluent UI ecosystem, there are icons related to bookmarks available in the Fluent UI System Icons collection. Specifically, there are icons such as the `Bookmark` icon and the `Bookmark Search` icon, which are designed to represent bookmarking features and search functionalities related to bookmarks.
But there's no a specific component as in your screenshot
